source/documentation/common/prereqs/ref-Host_Memory_Requirements.adoc

```diff
@@ -6,6 +6,6 @@
 // PPG
 // Install
 
-The minimum required RAM is 2 GB. The maximum supported RAM per VM in {hypervisor-fullname} is 4 TB.
+The minimum required RAM is 2 GB. The maximum supported RAM per VM in {hypervisor-fullname} is 16 TB.
```

Comment:
yeah, we increased the maximum memory per VM to 16 TB but I believe the previous setting (4TB) applied to all cluster levels whereas now on x86_64 it is 6 TB for cluster levels 4.2 to 4.5 and 16TB for cluster levels 4.6 and 4.7. I think we should at least mention that this is true for the latest cluster level (or to detail the per-cluster level value)